src/v1/components/surfaces/FloatingPanel/DraggableFloatingPanel.tsx

```diff
@@ -37,8 +43,19 @@ const DraggableFloatingPanel: React.FC<DraggableFloatingPanelProps> = ({
       bounds={bounds}
       enableResizing={false}
       dragHandleClassName="drag-handle"
+      default={{ x: defaultPosition.x, y: defaultPosition.y, width: 'auto', height: 'auto' }}
+      css={{
```

Comment:
can we use sx?

I'm not particualrly worried - strictly speaking I'm not worried about this specific use

But if posssible I'd rather not leave `css` props around for people to copy and expand usage

Reply:
It's not a MUI component so I cannot use the sx prop